src/stores/QueryStore.ts

```diff
@@ -95,10 +97,11 @@ export default class QueryStore extends Store<QueryStoreState> {
             routingProfile: {
                 name: '',
             },
-            customModelEnabled: false,
+            customModelEnabled: initialCustomModelStr != null,
```

Comment:
We open the custom model box if the `custom_model` url parameter is given. There is still something wrong with the css, because the box overlaps the 'Help' button and the 'Examples' Dropdown in this case. @karussell do you have an idea how to fix this?

Reply:
I use this url: http://localhost:3000/?custom_model={"distance_influence":70}
Then it looks like this (in Chrome, and the same problem in Firefox): 

![image](https://user-images.githubusercontent.com/17603532/188403469-d82a0cd7-f211-4d84-b313-44add542c419.png)

When I close and open again the custom model box it no longer overlaps the buttons.

Reply:
> But does not happen for me in Firefox :D

oh no :( not this again. For me it's the same in Chrome and FF

Reply:
Yes, that fixed it!

Reply:
... well not completely, because the box remains blank when there is no url parameter after opening the box, but see latest commits.